lib/event_framework/event_processor_runner.rb

```diff
@@ -20,18 +20,19 @@ def call(error, tries)
     def initialize(
       processor_class:,
       domain_context:,
+      logger: Logger.new($stdout),
```

Comment:
Just curious, why did you choose `$stdout` over `STDOUT`?

Also thought of this slight reason to prefer `STDOUT`:

```
$ irb
$irb(main):001:0> $stdout_typo
=> nil
irb(main):002:0> STDOUT_TYPO
Traceback (most recent call last):
        4: from /Users/odin/.rubies/ruby-2.6.2/bin/irb:23:in `<main>'
        3: from /Users/odin/.rubies/ruby-2.6.2/bin/irb:23:in `load'
        2: from /Users/odin/.rubies/ruby-2.6.2/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        1: from (irb):2
NameError (uninitialized constant STDOUT_TYPO)
```

Reply:
Standardrb enables the [Style/GlobalStdStream](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GlobalStdStream) cop by default 🤷‍♂️